src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialNewFilesPageObject.java

```diff
@@ -211,4 +213,8 @@ public String getImageKey(int itemNumber) {
     return imageKey;
   }
 
+  public String getNewFilesSpecialPageTitle() {
```

Comment:
this method is redundant, you can call field directly if you change it to public